### PR TITLE
BUcash release notes for version 1.5.0.0

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: doc/release-notes-bucash.md [hunk 1]
```diff
@@ -31,531 +30,230 @@ If you are upgrading from a release older than 1.1.2.0, your UTXO database will
...
-- Cashlib: create a shared library to make creating wallets easier (experimental, this library factors useful functionality out of bitcoind into a separate shared library that is callable from higher level languages. Currently supports transaction signing, additional functionality TBD)
-- Improve QA machinery (travis mainly)
-- Port Hierarchical Deterministic wallet (BIP 32)
-- add space-efficient mining RPC calls that send only the block header, coinbase transaction, and merkle branch: getminingcandidate, submitminingsolution
+- Implementation of November 2018 upgrades feature (see the [specification](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md) for more details)
+- CTOR, CDSV, CLEAN_STACK, FORCE_PUSH, 100 byte TXN SIZE (as a sublist)
+- Add configuration parameters to allow miners to specify their [BIP135](https://github.com/bitcoin/bips/blob/master/bip-0135.mediawiki) votes. See this [guide]() from more details
+- Multithreaded transaction admissio to the mempool (ATMP)
```
Comment (gandrewstone) [inline doc/release-notes-bucash.md:52]: admission (forgot last n)

Comment (sickpig) [reply doc/release-notes-bucash.md:52]: thanks will fix

Comment (sickpig) [reply doc/release-notes-bucash.md:52]: done

File: doc/release-notes/release-notes-bucash1.5.0.0.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+- Implementation of November 2018 upgrades feature (see the [specification](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md) for more details)
+- CTOR, CDSV, CLEAN_STACK, FORCE_PUSH, 100 byte TXN SIZE (as a sublist)
+- Add configuration parameters to allow miners to specify their [BIP135](https://github.com/bitcoin/bips/blob/master/bip-0135.mediawiki) votes. See this [guide]() from more details
+- Multithreaded transaction admission to the mempool (ATMP)
+- Parallelize message processing
+- Fastfilters: a faster than Bloom Filter probabilistic data structure
+- Various improvements to the Request Manager
+- Add tracking of ancestor packages and expose ancestor/descendant information over RPC
+- Remove trickle login in dealing with transactions INV
+- Implement shared lock semantics for the UTXO
```
Comment (ptschip) [inline doc/release-notes/release-notes-bucash1.5.0.0.md:58]: -Remove trickle login ...
+Remove trickle logic ...

**Pull request merged**